README.md

```diff
@@ -93,23 +113,9 @@ This collection can be generated using the [content_builder](https://github.com/
 
 This tool can generate the api specification files and use them to generate the modules. The developer can use the [api specification](https://github.com/ansible-collections/amazon.cloud/api_specifications) files and the [modules.yaml](https://github.com/ansible-collections/amazon.cloud/modules.yaml) file hosted in this repository. The path to these files should be provided as input to the content builder tool to generate the schema and modules. To get more details on the command and input arguments please refer to the tool's [README](https://github.com/ansible-community/ansible.content_builder#resource-module-scaffolding-generated-using-openapi-based-json).
 
-This collection is tested using GitHub Actions. To know more on testing, refer to [CI.md](https://github.com/ansible-collections/amazon.cloud/blob/main/CI,md).
-
-### See Also:
-
-* [Amazon.Cloud Collection Guide](https://docs.ansible.com/ansible/latest/scenario_guides/guide_amazon_cloud.html)
-* [Ansible Using collections](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html) for more details.
-
-
-## Known issues
-
-* Modules are only as good as the API and its schema. Documentation may not be complete for all the modules' options and suboptions.
-* Missing supportability for important AWS resources like, EC2 instance, RDS instance, EC2 volume, Elastic Load Balancer, RDS Snapshot, EC2 snapshot, etc.
-* Idempotency is a function of the API and may not be fully supported.
-* Missing server-side pagination. This may have severe impact on performance.
-* Name-based identification filtering to support desired state (idempotency) logic is absent. This limitation has made us to exclude several resources and reduce the numebr of modules. A client-side filtering will definitely have a large impact on performance.
-* Not all the resources support the available states. In practice this means that some resources cannot be updated or listed.
+## Testing
 
+This collection is tested using GitHub Actions. To know more on testing, refer to [CI.md](https://github.com/ansible-collections/amazon.cloud/blob/main/CI,md).
 
 ## Contributing to this collection
```

Comment:
I would suggest that we either replace the IRC links with forum links or, if not replace, then at least add forum links since that is the recommended communication channel.